duckduckgo/iOS
# Unified Home Row CTA

Task/Issue URL: https://app.asana.com/0/856498667320406/1153008287599818
Tech Design URL:
CC:

**Description**:
Made Home Row CTA include video.

**Steps to test this PR**:
1. Fresh install.
2. Home Row CTA should fire after onboarding.
3. Video should play. User should be able to replay the video.

Ensure following works:
- Tips are not shown till CTA is dismissed.
- Light&Dark themes.
- iOS 12 video.
- Dismissing works properly (CTA is not shown again).
- Day 3 reminder should be presented if video is not played to the end (CTA is dismissed before it's finished).


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
